.vscode/launch.json

```diff
@@ -1,29 +1,33 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Python: Run Django",
-            "type": "python",
-            "request": "launch",
-            "program": "${workspaceFolder}/manage.py",
-            "console": "integratedTerminal",
-            "args": [
-                "runserver",
-                "--noreload",
-                "--nothreading"
-            ],
-            "django": true
-        },
-        {
-            "name": "Python: Attach Debugger",
-            "type": "python",
-            "request": "attach",
-            "port": 5678,
-            "host": "0.0.0.0"
-        },
-    ]
-  }
\ No newline at end of file
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Is this file supposed to be committed to the repo?

Reply:
Yea, it is from c3g actually:

https://github.com/bento-platform/katsu/blob/develop/.vscode/launch.json

So I need to commit it or it goes away every time I switch my branch